### PR TITLE
Document k8s.io and kubernetes.io vanity url(s)

### PR DESCRIPTION
**Pull request by dims**
Make it easy for folks to discover our urls for various resources

### CODE REVIEW

**Comment by dims** [comment]
@ixdy @thockin PTAL

File: k8s.io/README.md [hunk 1]
```diff
@@ -3,6 +3,38 @@ Overview
...
+| Documentation | https://docs.k8s.io | https://docs.kubernetes.io |
+| Kubernetes examples | https://examples.k8s.io | https://examples.kubernetes.io |
+| Features repo | https://feature.k8s.io <br> https://features.k8s.io |  https://feature.kubernetes.io <br> https://features.kubernetes.io |
+| Install script | https://get.k8s.io | https://get.kubernetes.io |
+| Github organization| https://git.k8s.io | https://git.kubernetes.io |
+| Redirections | https://go.k8s.io | https://go.kubernetes.io |
+| Issues | https://issue.k8s.io <br> https://issues.k8s.io | https://issue.kubernetes.io <br> https://issues.kubernetes.io |
+| Main site | https://k8s.io |
+| Kops submit queue | https://kops.submit-queue.k8s.io |
+| [not used] | https://kubernet.es |
```
Comment (thockin) [inline k8s.io/README.md]: This can be removed from this list

**Comment by thockin** [comment]
otherwise LGTM

File: k8s.io/README.md (hunk 1 above)
Comment (dims) [reply k8s.io/README.md]: Done!

**Comment by dims** [comment]
@ixdy @thockin PTAL

File: k8s.io/README.md [hunk 2]
```diff
@@ -3,6 +3,37 @@ Overview
...
+
+|  | k8s.io | kubernetes.io |
+| --- | --- | --- |
+| APT downloads| https://apt.k8s.io | https://apt.kubernetes.io |
+| Changelog | https://changelog.k8s.io | https://changelog.kubernetes.io |
+| CI logs | https://ci-test.k8s.io | https://ci-test.kubernetes.io |
+| Git repo | https://code.k8s.io | https://code.kubernetes.io |
+| Community submit queue | https://community.submit-queue.k8s.io |
+| Contrib submit queue | https://contrib.submit-queue.k8s.io |
+| Downloads XML | https://dl.k8s.io | https://dl.kubernetes.io |
```
Comment (ixdy) [inline k8s.io/README.md]: It shows the s3-style XML if you don't specify a path, though it's meant to be used with a URI like https://dl.k8s.io/v1.7.5/kubernetes.tar.gz.

Maybe just drop the "XML"?

Comment (dims) [reply k8s.io/README.md]: Done!

**Pull request merged**